src/v2/Apps/PriceDatabase/Components/PriceDatabaseBenefits.tsx

```diff
@@ -0,0 +1,84 @@
+import {
+  Column,
+  GridColumns,
+  Text,
+  Image,
+  Separator,
+  Flex,
+  themeProps,
+} from "@artsy/palette"
+import React from "react"
+import { useMatchMedia } from "v2/Utils/Hooks/useMatchMedia"
+
+export const PriceDatabaseBenefits: React.FC = () => {
+  const isMobile = useMatchMedia(themeProps.mediaQueries.xs)
```

Comment:
Note that `useMatchMedia` isn't SSR friendly and instead we should use [@artsy/fresnel](https://github.com/artsy/fresnel)'s  `<Media>` component.